src/libclipper/src/redis.cpp

```diff
@@ -438,6 +449,22 @@ bool add_application(redox::Redox& redis, const std::string& appname,
   }
 }
 
+bool add_model_links(redox::Redox& redis, const std::string& appname,
+                     const std::vector<std::string>& model_names) {
+  if (send_cmd_no_reply<string>(
+          redis, {"SELECT", std::to_string(REDIS_APP_MODEL_LINKS_DB_NUM)})) {
+    for (auto model_name : model_names) {
```

Comment:
We're making multiple requests to Redis here instead of sending the whole set of `model_names` at once. This is because I couldn't find a way to add multiple elements to a set using the available `send_cmd_no_reply` infrastructure in one call. Let me know if there's a way to do this.

It's possible that no such way exists and sending a request for each new linked model is a bad practice. In this case, we could just have this REDIS_APP_MODEL_LINKS table store a mapping between app names and a string. The string would be a serialized list of model names (maybe with one of our predefined list delimiters). Then, whenever we want to fetch model links, we'd just deserialize the list. When adding new ones, we'd first fetch the list, add to it, and serialize it before setting the new value in the Redis table.

For now it's possible this isn't a big deal because we enforce that `model_names` has no more than 1 entry.